source/_posts/2017-11-27-Adding-Peek-and-Pop-to-your-View-Controllers.md

```diff
@@ -0,0 +1,145 @@
+---
+title:  Adding Peek and Pop to your View Controllers
+date: 2017-11-27 10:10:57
+tags: ios,peek,pop,3dtouch,forceTouchCapability
+authorIds:
+- dori
+categories:
+- iOS
+---
+
+With iOS 9, new iPhone models add a third dimension to the user interface.
+
+A user can now press your Home screen icon to immediately access functionality provided by your app.
+Within your app, a user can now press views to see previews of additional content and gain accelerated access to features.
+
+## Peek and Pop
+
+iOS 9 lets you configure view controllers for the user features of peek, which provides a preview of additional content when a user presses on a specified view, and pop, which commits to viewing that content and navigates to it.
+
+This interaction proceeds through three phases.
+
+1. Indication that content preview is available
+2. Display of the preview—known as a peek—with options to act on it directly—known as peek quick actions
+3. Optional navigation to the view shown in the preview—known as a pop
```

Comment:
Depending on the target audience an image/gif of these steps will aid the reader to understand the concept.